04-vue-cli/02-TheToaster/components/Toast.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div v-once class="toast" :class="toastClass">
+    <ui-icon v-once class="toast__icon" :icon="toastIcon" />
+    <span v-once>{{ message }}</span>
+    <!--<button v-if="closeButton" @click="$emit('close')">x</button>-->
+  </div>
```

Comment:
Обычно `v-once` нужен только, когда есть проблема с производительностью в текущем месте, и не потребуется обновлять эту часть дерева. Обычно это актуально только для больших статичных частей.

При этом `v-once` достаточно было бы только на корневом `div`. Он действует на всё поддерево узлов.

Из документации: 

> Once again, try not to overuse this pattern. While convenient in those rare cases when you have to render a lot of static content, it's simply not necessary unless you actually notice slow rendering - plus, it could cause a lot of confusion later. For example, imagine another developer who's not familiar with v-once or simply misses it in the template. They might spend hours trying to figure out why the template isn't updating correctly.

https://v3.vuejs.org/guide/component-edge-cases.html#cheap-static-components-with-v-once